specificatie/BRK-Bevragen/genereervariant/openapi.yaml

```diff
@@ -3859,10 +3858,10 @@ components:
       description: Hieraan wordt een Kadasterstuk of een Aangebodenstuk gekoppeld.
         Een stuk is een brondocument dat aanleiding geeft tot een wijziging van de
         gegevens in een basisregistratie.
-    AmbtelijkCorrectiegegevenID:
+    PortefeuilleId:
```

Comment:
Is het niet beter om dit gewoon Portefeuille te noemen? Bij een Id verwacht ik nl. alleen identificerende properties

Reply:
De properties die vallen onder deze resource zijn bedoeld om deze resource mee te kunnen identificeren.